src/status_im/contexts/communities/overview/view.cljs

```diff
@@ -73,25 +73,26 @@
                  (into #{} (map (comp :name second) channels-list)))
     :style     (style/channel-list-component)}
    (for [[category-id {:keys [chats name collapsed?]}] channels-list]
-     [rn/view
-      {:key       category-id
-       ;; on-layout fires only when the component re-renders, so
-       ;; in case the category hasn't changed, it will not be fired
-       :on-layout #(on-category-layout name category-id (int (layout-y %)))}
-      (when-not (= constants/empty-category-id category-id)
-        [quo/divider-label
-         {:on-press     #(collapse-category community-id category-id collapsed?)
-          :chevron-icon (if collapsed? :i/chevron-right :i/chevron-down)
-          :chevron      :left}
-         name])
-      (when-not collapsed?
-        [rn/view {:style {:padding-horizontal 8}}
-         (let [last-item-index (dec (count chats))]
-           (map-indexed
-            (fn [index chat]
-              ^{:key (:id chat)}
-              [channel-chat-item community-id chat (= index last-item-index)])
-            chats))])])])
+     (when (seq chats)
```

Comment:
`seq?`

Reply:
Thank you @flexsurfer for review.

Guideline suggests seq: https://github.com/bbatsov/clojure-style-guide#:~:text=is%20now%20nil-,Nil%20Punning,-Use%20seq%20as

Reply:
And [seq?](https://clojuredocs.org/clojure.core/seq_q) is just for checking if passed argument implements ISeq

Reply:
it depends is you are checking on nil or on empty?

Reply:
for nil check I guess, I could have used chats directly 😅

Reply:
anyway here should be seq i guess